ads/google/a4a/utils.js

```diff
@@ -301,6 +303,7 @@ export function googlePageParameters(a4a, startTime) {
           'top': win != win.top ? topWindowUrlOrDomain(win) : null,
           'loc': win.location.href == canonicalUrl ? null : win.location.href,
           'ref': referrer || null,
+          'bdt': domLoading > 0 ? startTime - domLoading : null,
```

Comment:
Do you worry about negative number?  Otherwise this could just be domLoading ?

Reply:
No. getPerformanceTiming returns 0 when the API is not supported, otherwise it would be a positive epoch time.

I guess using truthiness saves 2 bytes. Changed.